tests/Unit/Lti/Factory/Lti1p3RequestFactoryTest.php

```diff
@@ -26,6 +26,7 @@
 use OAT\Library\Lti1p3Core\Registration\Registration;
 use OAT\Library\Lti1p3Core\Registration\RegistrationRepositoryInterface;
 use OAT\SimpleRoster\Entity\Assignment;
+use OAT\SimpleRoster\Entity\LineItem;
```

Comment:
Maybe I'm missing it, but I don't see usage of LineItem entity in this TestClass

Reply:
You are correct. Removed